src/components/Header.js

```diff
@@ -0,0 +1,26 @@
+const React = require('react');
+const { NavLink } = require('react-router-dom');
+const { default: Styled } = require('styled-components');
+
+const internals = {};
+
+module.exports = () => {
+
+    const { Link } = internals;
+
+    return (
+        <div>
+            <h1>Strangeluv</h1>
+            <Link exact to='/'>Home</Link>
+            {' · '}
```

Comment:
This line causes a "Failed to compile" error for me:

>Line 15:13:  Curly braces are unnecessary here  react/jsx-curly-brace-presence

Reply:
I think the behavior might have changed when they fixed this issue: https://github.com/yannickcr/eslint-plugin-react/issues/2423 . Hmm...

If they are unnecessary, do you mind removing them?  If they are necessary and this is a false positive, can you downgrade eslint-plugin-react prior to the bug was introduced.

Reply:
It also might have an autofix worth trying!

Reply:
It looks like it's not valid because of the period.

```
 * there's no way to inject a whitespace into jsx without a container so this
 * will always be allowed.
 */
<App>{' '}</App>
<App>{'     '}</App>
```
([documentation](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md))
So I think you could technically do {' '}.{' '}?

Reply:
It's probably this bug that has a fix that hasn't been released yet: https://github.com/yannickcr/eslint-plugin-react/issues/2427

When this PR went-up it definitely compiled fine and passed the CI checks.  In the past couple days eslint-plugin-react has made several changes to this particular lint rule, so I expect they are working out some kinks.  I would recommend downgrading locally for now until things settle there.  Try `npm install --save-dev eslint-plugin-react@7.14.3`.

Reply:
Downgrading `eslint-plugin-react` didn't actually do it, but downgrading `eslint` did the trick for now.

Reply:
Huh, pretty odd!  Glad you worked around it for now.

Reply:
I ran into this same issue, even after upgrading to `v7.16.0` which was _supposed_ to fix it (This PR https://github.com/yannickcr/eslint-plugin-react/pull/2431)

Someone else commented that it actually didn't fix the issue and they opened a new one. Last comment was 3 hours ago https://github.com/yannickcr/eslint-plugin-react/issues/2454)

Reply:
I don't want to spend too much time on this. I put in a temp fix and comment about it by declaring a var for this
```
const divider = ' · ';
...
{divider}
```

Reply:
This part of the code is changed in Tyler's material branch anyway so we'll be fine